pkg/operator/ceph/csi/version.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+// CephCSIVersion represents the Ceph CSI version format
+type CephCSIVersion struct {
+	Major  int
+	Minor  int
+	Bugfix int
+}
+
+var (
```

Comment:
can you declare the variables in beginning and define a struct and its methods,
it good to keep struct and its methods in next to each other for better readability

Reply:
You want me to create an `interface` for the `CephVersion` structure?
Or do I get you wrong?

Reply:
no, I was saying declare all variables and then declare struct followed  by the methods it implements as it increases the readability

```go
var (
a int
b int
)

type abc struct{
}

func (a abc)supported(){
}
func  (a abc)Version(){
}
```

Reply:
OK. Swapped those two sections.